components/Auction/index.js

```diff
@@ -113,34 +113,53 @@ const HandleAuctions = () => {
     return (
       <div className={`${styles.auctionContainer} ${id}`} key={id}>
         <div className={styles.auctionSeller}>
-          <h2>Seller:</h2>
           <img
             className={styles.profilePhoto}
             src={`${BASE_IMAGE_URL}/${seller}/img.png`}
             onError={brokenImageHandler}
           />
         </div>
-        <div className={styles.auctionStats}>
-          <div className={styles.itemInfo}>
-            <h1>
-              {quantity} x {'  '}
+        <div className={styles.auctionStatsBidderWrapper}>
+          <div className={styles.auctionStats}>
+            <div className={styles.itemInfo}>
+              <h1>
+                {quantity} x {'  '}
+              </h1>
               <Image
                 className={styles.gemImage}
                 layout="fixed"
                 src="/assets/gem.png"
                 width={25}
                 height={25}
               />
-            </h1>
+            </div>
+            <div className={styles.currentStatus}>
+              <h2>Current Bid:</h2>
+              <h2>
+                <div className={styles.currentBidPrice}>{highest_bid}</div>
+              </h2>
+            </div>
           </div>
-          <div className={styles.currentStatus}>
-            <h2>
-              Current Bid:
-              <div className={styles.currentBidPrice}>{highest_bid}</div>
-            </h2>
+          <div className={styles.bidders}>
+            {bidders.map((bidder) => {
+              return (
+                <div
+                  className={styles.biddersImg}
+                  key={bidder}
+                  data-columns={getColumns(bidders.length)}
+                  title={bidder}
+                >
+                  <img
+                    className={styles.profilePhoto}
+                    src={`${BASE_IMAGE_URL}/${bidder}/img.png`}
+                    onError={brokenImageHandler}
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#browser_compatibility
can you please check this it seems like being deprecated
![image](https://user-images.githubusercontent.com/33935339/219962811-964f5ebb-d287-4964-8509-263b9cb98f8f.png)